Dependencies/protobuf-2.0.3/README.txt

```diff
@@ -1,12 +0,0 @@
-This directory includes binaries and embeddable source code from Google Protocol Buffers
-http://code.google.com/p/protobuf/ 
-http://code.google.com/p/protobuf-c/ 
-
-The binaries were build from unmodified protobuf 2.0.3 and protobuf-c 0.6 sources, targeted at:
-    - Mac OS X 10.5
-
-The source (see src/) directory, is an extraction of the protobuf-c runtime library. It
```

Comment:
The protobuf sources were previously modified with some fixes, we should double check if these modifications are still needed in the new version.

Reply:
Do we have to contact "Plausible Labs" for this? or protocol buffer maintainers?